### ISSUE TITLE
[Feature Request] v-select aria roles

### ISSUE THREAD

**Issue by ksenechkaP** [vuetifyjs/vuetify#3999]
### Problem to solve
Select options are not accessible for screen reader users. 

### Proposed solution
Please add option for aria-label, so  screen readers can more easily navigate it.

**Comment by joestrouth1** [comment]
Related: 

[v-menu](https://next.vuetifyjs.com/en/components/menus) seems difficult to navigate with keyboard & screenreader. 

When focused, it is announced as a regular push button and gives no indication of 'collapsed' state. After activating with space, I receive no indication that it is open or of the menu options in the list. Suggest comparing with [bootstrap-vue's dropdown](https://bootstrap-vue.js.org/docs/components/dropdown) as it is fairly intuitive.

The clickable-image examples on the v-menu page are skipped altogether when reading/tabbing through and I could not successfully toggle them.

**Comment by johnleider** [comment]
This is something we are definitely going to do, just didn't make the cut for v1.3

**Comment by TuntematonSotilas** [comment]
Same thing on v-autocomplete and v-combobox :
My screen reader have no aria to read 
The navigation on the list says only "empy" on each element of the list selected
Tested with NVDA screen reader (https://www.nvda-fr.org/)

**Comment by TuntematonSotilas** [comment]
Hello in Autocomplete and Combobox components, I can't see any of aria attributes describe in W3C :
https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

It would be nice to have have aria attributes in the components Autocomplete/Combobox

On the combobox container :
• role="combobox"
• aria-haspopup="listbox"
• aria-expanded="true/false" (compute ?)
• aria-owns="IDREF"

On the Input :
• aria-autocomplete="list"
• aria-activedescendant="IDREF" : (compute ?)

On the listbox Popup :
• role="listbox"
• aria-labelledby="IDREF" : (compute ?)
• role="option"

**Comment by KaelWD** [comment]
We know, that's why this issue is still open.

**Comment by suswinther** [comment]
Hi guys, when do you think this will be available?

**Comment by jacekkarczmarczyk** [comment]
I'll try to take a look in next few days, but can't promise anything

**Comment by ZeevGerstner** [comment]
Hay, did you add accessibility options (aris,role etc) to v-complete component?

**Comment by johnleider** [comment]
This was resolved with #7385 .

Thank you to everyone for your patience on this.
